dist/backdraft.js

```diff
@@ -1214,42 +1258,68 @@ _.extend(Plugin.factory, {
     // @col: The column from the column manager corresponding to the column we're creating
     //   filtering wrappers for.
     _createFilteringWrappers: function(head, col) {
+      var table = this;
+      table.activeMenu = null;
```

Comment:
better that its on the table now.. but how is it different from filterActive and can it be named activeFilterMenu at least?